client/src/components/signUpForm/SignUpForm.js

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { useForm } from "react-hook-form";
+import SignUpButton from "../signUpButton/SignUpButton";
+import postAccount from "../../apiAccount/postAccount";
+const SignUpForm = () => {
+  const { register, handleSubmit, errors } = useForm();
+
+  const onSubmit = async (data) => {
+    await postAccount(data);
+    console.log(data);
+  };
+
+  return (
+    <Form>
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <section>
+          <label htmlFor="name">Braut</label>
+          <input
+            id="name"
+            autoComplete="off"
+            name="DeinName"
+            placeholder="Name"
+            ref={register({ required: true })}
+          />
```

Comment:
You could wrap the label around the input field. If you do so and click on the label description, the cursor will automatically move into the input field. This will give you some extra UX-points :) 
```suggestion
          <label htmlFor="name">Braut
          <input
            id="name"
            autoComplete="off"
            name="DeinName"
            placeholder="Name"
            ref={register({ required: true })}
          />
          </label>
```